src/helpers/customForms.ts

```diff
@@ -644,7 +644,7 @@ const getFieldValidation = (question: FormQuestionRead, t: typeof useT, framewor
             const { demographics } = value.length > 0 ? value[0] : {};
             if (demographics == null) return true;
 
-            return calculateTotals(demographics, framework).complete;
+            return calculateTotals(demographics, framework).counts;
```

Comment:
I don't believe this is correct. this `test` method is supposed to return a boolean indicating if the error message should display or not. I updated this line earlier today and I'm pretty sure `complete` is correct.

FWIW, this line as it was this morning is probably what was causing the error with save & continue mentioned in the ticket.